tests/helpers.ts

```diff
@@ -43,9 +37,8 @@ export function nextMicroTick(): Promise<void> {
 }
 
 export async function nextTick(): Promise<void> {
-  return new Promise(function (resolve) {
-    setTimeout(() => scheduler.requestAnimationFrame(() => resolve()));
-  });
+  await new Promise((resolve) => scheduler.requestAnimationFrame(resolve));
+  await new Promise((resolve) => setTimeout(resolve));
```

Comment:
Interesting... That's not how it has been implemented in Odoo. In Odoo, we wait for a setTimeout before the nextAnimationFrame. I believed this was correct when we wrote it. We may need to talk about that to ensure that we have the correct implementation in both projects.